packages/lesswrong/components/posts/EALargePostsItem.tsx

```diff
@@ -182,7 +188,8 @@ const EALargePostsItem = ({
             </div>
           </div>
           <div className={classes.postListItemPreview}>
-            {post.contents?.plaintextDescription}
+            {postContents?.plaintextDescription ?? post?.contents?.plaintextDescription}
+            {loading && <Loading />}
```

Comment:
I'm slightly worried about this introducing loading spinners when it already has the data. Consider doing `{!plaintextDescription && loading &&` (with an extracted variable)